### PR TITLE
Add trigger to build userdocs on release

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Adds trigger for https://github.com/weaveworks/profiles/issues/242

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

**Review by Callisto13 — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -74,6 +74,12 @@ jobs:
           echo 'release_notes<<EOF' >> $GITHUB_ENV
           cat docs/release_notes/${{ steps.prep.outputs.VERSION }}.md >> $GITHUB_ENV
           echo 'EOF' >> $GITHUB_ENV
+      - name: Build Profiles Userdocs
+        uses: peter-evans/repository-dispatch@0e8ca8c8a5ca5e28d25af2c27fc5aa40f16cffca
+        with:
+          token: ${{ secrets.WEAVEWORKS_PCTL_USERDOC_DISPATCH_TOKEN }}
+          event-type: pctl-release
+          repository: weaveworks/profiles-userdocs
```
Comment (aclevername) [inline .github/workflows/release.yaml:82]: what does this do?

Comment (Skarlso) [reply .github/workflows/release.yaml:82]: Triggers a [repository_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#repository_dispatch) event.

Comment (aclevername) [reply .github/workflows/release.yaml:82]: what does that mean? We have some API webhook running in the `profiles-userdoc` repo that recieves this request?

Comment (Skarlso) [reply .github/workflows/release.yaml:82]: Read the docs, Jake. :D

Comment (Skarlso) [reply .github/workflows/release.yaml:82]: It's all Github. There is nothing running anywhere. It's a trigger type for actions. :) Respectively in profiles-userdocs there is a trigger like this:

```yaml
on:
  repository_dispatch:
    types: [pctl-release, profiles-release]
```

Which will react to the triggered event with the right keywords.

**Review by aclevername — APPROVED** [review]

**Comment by Himangini** [comment]
Closing this off since we no longer need this, sorry :(

**Pull request closed without merging**